README.md

```diff
@@ -50,6 +50,11 @@ hdcli tracker init
 hdcli tracker run
 ```
 
+```bash
+# Send information about your project manifest files
+hdcli report ingestion
```

Comment:
Down to call it ingestion everywhere except top level user experience. `hdcli report generate` or `submit` or `capture` would be better imo. 

Def open to thoughts / feedback though.

Reply:
renamed to `hdcli report generate`